libs/providers/flagd/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@openfeature/flagd-provider",
+  "version": "0.0.1",
+  "type": "commonjs",
+  "scripts": {
```

Comment:
You may have to add a `peerDependencies` block as @beeme1mr did in this commit https://github.com/open-feature/node-sdk-contrib/pull/64/commits/75dc1c5ccc06bcaa10273bad8b80ed8b1ea1bb50
```json
  "peerDependencies": {
    "@openfeature/nodejs-sdk": "^0.0.1"
  }
```

Reply:
added and removed other dependencies